content/pt-br/docs/tasks/access-application-cluster/configure-dns-cluster.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Configurar DNS para um Cluster.
+weight: 120
+content_type: concept
+---
+
+<!-- overview -->
+Kubernetes oferece um complemento de DNS para os clusters, que a maioria dos ambientes suportados o tem habilitado por padrão. Na versão do Kubernetes 1.11 e posterior, o CoreDNS é recomendado e instalado por padrão com o kubeadm.
+
+<!-- body -->
+Para mais informações sobre como configurar o CoreDNS para um cluster Kubernetes, veja o [Personalização do Serviço de DNS](/docs/tasks/administer-cluster/dns-custom-nameservers/). Um exemplo que demonstra como usar o DNS do Kubernetes com o kube-dns, veja o [Plugin de exemplo para DNS](https://github.com/kubernetes/examples/tree/master/staging/cluster-dns).
```

Comment:
"consulte" instead of "veja o"

Reply:
Done.

Reply:
Minor: See the "veja o" in the first sentence.